sdk/core/src/retry_policy.rs

```diff
@@ -0,0 +1,123 @@
+use tokio::time::{sleep, Duration};
```

Comment:
It's a real shame that we'll end up relying on tokio, but I guess that's the way it will have to be...

Reply:
I've removed the dependency because I've implemented a `sleep` future by hand (it just spawns a thread that wakes the reactor after sleeping). For the time being I think it is good enough.